README.md

```diff
@@ -54,18 +54,17 @@ winget install Microsoft.PowerToys -s winget
 
 ### Other install methods
 
-There are [community driven install methods](./doc/unofficialInstallMethods.md) such as Chocolatey and Scoop. If these are your preferred install solutions, you can find the install instructions there.
+There are [community-driven install methods](./doc/unofficialInstallMethods.md) such as Chocolatey and Scoop. You can find the install instructions if these are your preferred install solutions.
```

Comment:
I don't like how the second adjustment here sounds.  It sounds awkward.

Reply:
i guess, changed it.